server/logger.config.ts

```diff
@@ -0,0 +1,53 @@
+import { configure, getLogger } from 'log4js';
```

Comment:
I think that this entire file should be a class.
pass the `logger.level` in the constructor and initiate the `configure` method in too.
hold a property like this:
```typescript
log = getLogger();
```
then, in your `server.ts` do something like:
```typescript
logger = new Logger('debug')
logger.log.debug(...)
```